app/presenters/keyword_presenter.rb

```diff
@@ -12,6 +12,18 @@ def matching_adword_urls
     return unless defined?(filter_params[:adwords_url_contains]) && filter_params[:adwords_url_contains]
 
     keyword.ads_top_urls.select { |item| item.downcase.include?(filter_params[:adwords_url_contains].downcase) }
```

Comment:
```suggestion
    adwords_url_contains = filter_params[:adwords_url_contains].downcase
    keyword.ads_top_urls.select { |item| item.downcase.include?(adwords_url_contains) }
```

Sorry I missed that one in the previous PRs 🙈

Reply:
(for performance reason, let's not downcase the same string multiple times in a loop)